frontend/src/browser/users/EditUser.js

```diff
@@ -500,15 +506,15 @@ export class EditUser extends Component {
                       name="email"
                       type="email"
                       component={this.renderInput}
-                      label={`${formatMessage(messages.email)}*`}
+                      label={`${formatMessage(messages.email)}${roles.includes(rolesList.get('LECTOR').id) ? '' : '*' }`}
```

Comment:
ten role check by som vytiahol hore pred return, kedze sa to opakuje tak do premennej a tu potom dole pouzit. napr. 

    const isLector = ...